packages/@react-stately/numberfield/src/useNumberFieldState.ts

```diff
@@ -33,24 +26,49 @@ export interface NumberFieldState {
   maxValue: number,
   value: number,
   inputValue: string,
-  textValue?: string,
-  currentNumeralSystem?: string
+  textValue?: string
+}
+
+// I don't know how to remove this one so it's not duplicated and not being imported from aria, should it be in react-types? shared?
```

Comment:
Yeah should go in types. I guess shared is fine? Otherwise we'd need to create a whole new i18n types package... we do have a `locale.d.ts` file in there already 🤔